lib/utils/client-helpers.ts

```diff
@@ -1,52 +1,47 @@
+export type ChainModifiers =
+  | 'allLocales'
+  | 'noLinkResolution'
+  | 'noUnresolvableLinks'
+  | unknown
+  | undefined
+
+export type ChainOption<Modifiers extends ChainModifiers = undefined> = {
+  withoutLinkResolution: unknown extends Modifiers
+    ? boolean
+    : 'noLinkResolution' extends Modifiers
+    ? true
```

Comment:
Not sure what to say about this indentation. Looks like prettier is being creative. -.-